src/utils.jl

```diff
@@ -163,10 +190,10 @@ Base.copy(q::Q) where {Q<:AbstractQuantity} = new_quantity(Q, copy(ustrip(q)), c
 
 Remove the units from a quantity.
 """
-ustrip(q::AbstractQuantity) = q.value
+@inline ustrip(q::AbstractQuantity) = q.value
 ustrip(::AbstractDimensions) = error("Cannot remove units from an `AbstractDimensions` object.")
 ustrip(aq::AbstractArray{<:AbstractQuantity}) = ustrip.(aq)
```

Comment:
This type parameter dispatch looks much more innocuous than the other one since you own `ustrip`. Although, even here you might be able to get away without the `<:AbstractQuantity` dispatch and just handle all `AbstractArray`s, so long as `ustrip` has a generic no-op fallback for reals?

Reply:
(I was commenting on line 195, and I realize that this wasn't really modified by the PR, so apologies for the slightly off topic note)

Reply:
Fixed in #37